internal/cli/iam/organizations/apikeys/accesslists/create.go

```diff
@@ -71,12 +71,26 @@ func (opts *CreateOpts) Run() error {
 		return err
 	}
 
-	r, err := opts.store.CreateOrganizationAPIKeyAccessList(opts.ConfigOrgID(), opts.apyKey, req)
+	useWhitelist, err := shouldUseWhitelist(opts.store)
 	if err != nil {
 		return err
 	}
 
-	return opts.Print(r)
+	var result *atlas.AccessListAPIKeys
+	if useWhitelist {
+		r, e := opts.store.CreateOrganizationAPIKeyWhitelist(opts.ConfigOrgID(), opts.apyKey, fromAccessListAPIKeysReqToWhitelistAPIKeysReq(req))
+		if e != nil {
+			return e
+		}
+		result = fromWhitelistAPIKeysToAccessListAPIKeys(r)
```

Comment:
[nit] I don't know if we try and remove `if-else` structures in go, but would read cleaner to me as an 
```
if {
    .. do something
    ...return result
  }
  .. do something else
  ... return result
}
```

Reply:
the reason is the way it is it's because we join the branch later to `return opts.Print(result)` I could duplicate the line if you think it would read better.

Reply:
Yeah that's how I would have written it, with the `return opts.Print(result)` in both cases. It's just a nut/style thing - your call if you prefer.

Reply:
a mix of @colm-quinn's point and my own, `opts.Print` doesn't care about type do you need to make result typed and add to the `if` and do you need `fromWhitelistAPIKeysToAccessListAPIKeys`?